### ISSUE TITLE
WebDNN doesn't use WebGL on Vega GPU on Safari tech preview 48 (11.2 beta) although supports WebGL2 also..

### ISSUE THREAD

**Issue by oscarbg** [mil-tokyo/webdnn#767]
Hi,
due to WebGPU untestable for me albeit having it enabled tests going  fallback to webassembly and seeing there is Webgl support I have enabled WebGL2 under experimental features in Safari tech preview 48 (11.2) and check webglreport I have both WebGL and WebGL2 support..
anyway visiting https://mil-tokyo.github.io/webdnn/ I get :
This Browser
WebGPU | WebGL | WebAssembly/asm.js | Fallback
-- | -- | -- | --
Supported | Not supported | Supported | Supported

what WebGL extensions/features WebDNN needs that Safari tech preview with WebGL 2 enabled lacks?
Is site checking a list of WebGL supported browsers and disabling it although Safari tech preview could work?
thanks..

**Comment by Kiikurage** [comment]
No, it's not because of availability of WebGL2, but the feature of reading texture values as float32 value, which is required to use WebGL texture buffers as float32 matrix buffers for GPGPU. Unfortunately, this feature hasn't supported by Safari.  There are some workarounds like [this](https://github.com/mikolalysenko/glsl-read-float). However we don't use them because of performance issue.

**Comment by milhidaka** [comment]
WebDNN has been completely rewritten as **v2.0.0-alpha**, a browser-based ONNX inference runtime (load an ONNX model directly in the browser; no Python conversion step). The v1 Keras / Chainer / TensorFlow / Caffe converters and the old backends that this issue refers to no longer exist, so this report is no longer actionable against the current codebase.

Closing as *not planned*. Thanks for the report!
